### PR TITLE
Movesense integration

### PR DESCRIPTION
**Pull request by bardram**
This PR contains all the changes from PR #229 PLUS the following:

* support for the Movesense Device using the new carp_movesense_package 
* showing Movesense HR data in the HR dataviz (in addition to the Polar HR data)
* a test protocol showing how to use the movesense device (in local deployment mode)
* more robust handling if some local resources (like informed consent) isn't available
* better handling of connecting to Bluetooth devices
* fix of #157

### CODE REVIEW

**Comment by bardram** [comment]
You should ignore the GitGuardian Security Checks - the "secrets" are a couple of API Key for the weather and air quality APIs, so not really secrets as such. And - they have been deleted again.

**Comment by LarsRefsgaard** [comment]
as long as it's not building on ios we can't really accept the pr

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.5.0
+
+* support for running app in local deployment mode in order to debug a protocol before uploading it to CAWS
+
```
Comment (SlimShadyIAm) [inline CHANGELOG.md:4]: we should probably update this more

File: lib/ui/cards/distance_card.dart [hunk 2]
```diff
@@ -131,7 +131,7 @@ class _DistanceCardState extends State<DistanceCard> {
         enabled: true,
         touchTooltipData: LineTouchTooltipData(
           fitInsideHorizontally: true,
-          tooltipBgColor: Colors.grey.withOpacity(0.1),
+          // tooltipBgColor: Colors.grey.withOpacity(0.1),
```
Comment (SlimShadyIAm) [inline lib/ui/cards/distance_card.dart:134]: can this be removed?

**Review by SlimShadyIAm — CHANGES_REQUESTED** [review]

**Review by SlimShadyIAm — APPROVED** [review]

**Pull request merged**